test/e2e/network/dual_stack.go

```diff
@@ -205,6 +206,165 @@ var _ = SIGDescribe("[Feature:IPv6DualStackAlphaFeature] [LinuxOnly]", func() {
 
 		assertNetworkConnectivity(f, *serverPods, *clientPods, "dualstack-test-client", "80")
 	})
+
+	ginkgo.It("should create service with cluster ip from primary service range [Feature:IPv6DualStackAlphaFeature:Phase2]", func() {
+		serviceName := "defaultclusterip"
+		ns := f.Namespace.Name
+		jig := e2eservice.NewTestJig(cs, serviceName)
+
+		defaultIPFamily := v1.IPv4Protocol
+		if framework.ClusterIsIPv6() {
```

Comment:
my apologies, but now this is a method `framework.TestContext.ClusterIsIPv6()`
https://github.com/kubernetes/kubernetes/pull/82060/files